contentScript.js

```diff
@@ -298,6 +299,7 @@ const repFun = () => {
         dispatchClicksAndBook();
       } else {
         $("[formcontrolname=pincode]").val(first_5_pin_digits);
+        $("[formcontrolname=pincode]")[0].dispatchEvent(new Event("input", { bubbles: true }));
```

Comment:
If you want to trigger the input event then this should come after the event declaration below.
Also, why not use jquery to trigger the event?
```suggestion
        $("[formcontrolname=pincode]").trigger('input');
```

Reply:
We want bubbles!